serverless.yml

```diff
@@ -50,6 +50,7 @@ functions:
       BINARY_CONTENT_TYPES: application/pdf
       GSSO_URL: ${ssm:/dlo-bonus-scheme/${self:provider.stage}/gsso-url}
       GSSO_TOKEN_NAME: ${ssm:/dlo-bonus-scheme/${self:provider.stage}/gsso-token-name}
+      GSSO_DOMAIN_NAME: ${ssm:/dlo-bonus-scheme/${self:provider.stage}/gsso-domain-name}
```

Comment:
Does this need to be set in a `*.env` file for it to pass locally?

Reply:
Yes, it does - I have a `.env.test.local` and I have it set in the CircleCI environment variables. However it could probably live in `.env.test` as week (as could `GSSO_TOKEN_NAME` probably).

Reply:
👍 I guess we'd benefit from at least an empty placeholder in `.env.test`

Reply:
You can't have the empty placeholder as it doesn't overwrite in the later file

Reply:
Don't think I was clear - I meant something like [this](https://github.com/LBHackney-IT/bonuscalc-frontend/blob/develop/.env.test#L15)